InnoTopicWebsite/src/app/cv-page/topics-graph/topics-graph.component.ts

```diff
@@ -352,6 +352,8 @@ export class TopicsGraphComponent implements OnInit {
       width = +svgRootElement.attr("width"),
       height = +svgRootElement.attr("height");
 
+      svgRootElement.selectAll('*').remove();
```

Comment:
Is this a hack or ?
What is the root cause?

Reply:
It is not a hack. You have to remove old elements before making the graph. The catch here is that the topics graph was getting loaded twice. 
1. When the application runs for the first time
2. When the application finds that `isPrint` is `true`, and by the time it starts to render it already has data/elements from the first step. Resulting in multiple objects to render.